tests/fixtures/TagLib.Tests.FileFormats/OpusFormatTest.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using NUnit.Framework;
+using TagLib;
+
+namespace TagLib.Tests.FileFormats
+{
+    [TestFixture]
+    public class OpusFormatTest : IFormatTest
+    {
+        private static string sample_file = "samples/sample.opus";
```

Comment:
You didn't include the sample.opus file in the PR. Do you still have the file by any chance? Can you add it to the PR?